### ISSUE TITLE
[obsolete documentation] YAML Front matter tags for per-episode picture and player skin are wrong

### ISSUE THREAD

**Issue by j-levy** [jekyll-octopod/jekyll-octopod.github.io#1]
On page "Usage" : https://jekyll-octopod.github.io/usage
The YAML front matter tag in order to put a picture for a specific episode is wrong.
For a picture named `episode-cover0.jpg`located in the `/img/` folder, instead of being :
```yaml
episode-cover: "/img/episode-cover0.jpg"
```
It should be instead:
```yaml
image: "episode-cover0.jpg"
```

Also, on page Configuration: https://jekyll-octopod.github.io/configuration
when trying to specify the player theme (for example, default-dark) with 
```yaml
player_theme: default-dark # default | default-dark | legacy | minimal | republica
```
Nothing happens. But if this is placed inside the YAMl front matter of a post markdown file (inside the _posts folder), it works correctly.

The documentation should be updated accordingly. 
Also, if possible, the command that generates the post file `octopod episode` should be modified to produce a valid file with correct fields in the first place, but I know that this is not the correct repository to report this. (just sayin' though).

**Comment by haslinger** [comment]
I completely missed this issue, because Github didn't notify me.
Will look into this the next days for fixes.
Tasks for me:
- [x] deploy the updated github page
- [x] check if the config is now working properly after my fix (player theme should be on site, not page level, that was a code bug, not a documentation bug)
- [x] create a new version for the gem 
- [x] check the output of octopod episode (I personally never used that ever, didn't even remember it to exist :see_no_evil:)

**Comment by haslinger** [comment]
Fixed in version 0.9.15 of the gem.
